be/src/olap/tablet.cpp

```diff
@@ -725,7 +727,23 @@ const uint32_t Tablet::calc_compaction_score(CompactionType compaction_type) con
     }
 }
 
-const uint32_t Tablet::_calc_cumulative_compaction_score() const {
+const uint32_t Tablet::_calc_cumulative_compaction_score() {
+#ifndef BE_TEST
+    std::string current_policy = "";
+    {
+        std::lock_guard<std::mutex> lock(*config::get_mutable_string_config_lock());
```

Comment:
It's better to check the config at the beginning of each `_compaction_tasks_producer_callback` loop,
instead of check it for each tablet.

Reply:
> It's better to check the config at the beginning of each `_compaction_tasks_producer_callback` loop,
> instead of check it for each tablet.

Thanks, it sounds reasonable.